contrib/grpc_metrics/src/main/java/io/opencensus/contrib/grpc/metrics/RpcViews.java

```diff
@@ -52,7 +53,26 @@
           RpcViewConstants.RPC_SERVER_UNCOMPRESSED_REQUEST_BYTES_VIEW,
           RpcViewConstants.RPC_SERVER_UNCOMPRESSED_RESPONSE_BYTES_VIEW,
           RpcViewConstants.RPC_SERVER_STARTED_COUNT_CUMULATIVE_VIEW,
-          RpcViewConstants.RPC_SERVER_FINISHED_COUNT_CUMULATIVE_VIEW);
+          RpcViewConstants.RPC_SERVER_FINISHED_COUNT_CUMULATIVE_VIEW,
```

Comment:
I would not put the new views in the same list, instead I will create a new list and have a different method that installs all the new views.

Then mark the old one as deprecated and suggest to use the new one, also explaining what it will happen (user may no longer have data in the old metrics) and suggesting that user may decide to still register the old views while they will deprecate the old views.

Reply:
Sounds good, I added a new method `registerAllGrpcViews` that registers all the new views. While `registerAllViews` will register both new and old views.

Reply:
I think the old one should register just the old views.

Reply:
Done, though the method name registerAllViews could be a little confusing while it only registers the deprecated views.